kolibri/plugins/user_profile/assets/src/constants.js

```diff
@@ -1,5 +1,5 @@
 export const ComponentMap = {
-  PROFILE: 'ProfilePage',
-  PROFILE_EDIT: 'ProfileEditPage',
+  PROFILE: 'PROFILE',
+  PROFILE_EDIT: 'PROFILE_EDIT',
   NEW_PASSWORD: 'NewPasswordPage',
```

Comment:
this should be deleted as it's not in use anymore

Reply:
Are you referring to line 4? In that case this constant is being used in some of the files.

Reply:
Thank you, @jredrejo.

@camperjett Some of our routes don't have `name` and only have `component` as an identifier. For such routes, we usually create these maps so that we don't need to reference components' names directly from `$router.getRoute`. However, `PROFILE` and `PROFILE_EDIT` routes already have names. Therefore we don't need to use this map anymore at all (no need to map a route name to the same route name).

To sum it up together with @jredrejo's suggestion, I'd suggest removing this map completely and then simply using  `$router.getRoute('PROFILE_EDIT')`, for example.

Reply:
@camperjett @jredrejo Neither comment is blocking though I'd say. This fixes the bug, and our feedback is related to some cleanup of legacy routing things.

@camperjett You're welcome to push those changes if you'd like, leaving it open for some time. Thanks for cooperation.

Reply:
PTAL